### DISCUSSION TITLE
USB WiFi connector not listed

### DISCUSSION THREAD

**Discussion by manuelsirianni** [nathanmarlor/foxess_modbus#416] [Support]
Hi, i've just started with a FoxEss H1 Inverter with a new solar plant and i've installed your Integration via HACS.
Unfortunately i have a different connector from your list, named FoxESS Smart WiFi USB.
I found it on this website: https://www.estg.eu/en-gb/foxess-smart-wifi-usb/

I tried configuring it as "Other" leaving the default info as:
USB Serial Device: /dev/ttyUSB0
Inverter Slave ID: 247

I'm sure that 247 is correct because i looked at inverter settings under RS485 menu.

I receive this error:
Unable to connect to your adapter. Ensure that the adapter is properly configured (see the setup link above) and the connection details are correct, then try again. Details: [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'; [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'; [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'; [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'

What can i do to fix the issue to configure the inverter in the right way?

Thanks
Regards

**Comment by canton7** [comment]
https://github.com/nathanmarlor/foxess_modbus/discussions/278

**Reply by manuelsirianni** [reply]
My stick is the 2.0 not the 3.0.
It's the same? So i can't get info from that?

Is it possibile to leave this stick and add also a RS485?
Thanks

**Reply by calum-mcfarlane** [reply]
Not only is it possible, it's the only way of doing it - the USB port for the Wifi stick doesn't work for any other purpose, you need to connect to the serial port.
Have a read of [this page](https://github.com/nathanmarlor/foxess_modbus/wiki/Modbus-Wiring-Guide) of the wiki, it helps to explain.
